<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextViewer</class>
 <widget class="QWidget" name="TextViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TEXT VIEWER</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="findStr">
        <property name="toolTip">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFind">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Find</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnReload">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Reload</string>
        </property>
        <property name="icon">
         <iconset resource="../rtkconv_qt/rtkconv_qt.qrc">
          <normaloff>:/buttons/reload.bmp</normaloff>:/buttons/reload.bmp</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRead">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Read...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpt">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Option...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>&amp;Close</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rtkconv_qt/rtkconv_qt.qrc"/>
 </resources>
 <connections/>
</ui>
